library/MedEx/API.php

```diff
@@ -1521,78 +1521,66 @@ function SMS_bot_list() {
         } else {
             echo 'up';
         } ?> menu_arrow" style="position: fixed; left: 5px; top: 5px; z-index: 1099;" id="patient_caret" onclick='toggle_menu();' aria-hidden="true"></i>
-        <div id="hide_nav" style="<?php if ($setting_bootstrap_submenu == 'hide') {
+        <div id="hide_nav" class="mt-3" style="<?php if ($setting_bootstrap_submenu == 'hide') {
             echo "display:none;"; } ?>">
-            <nav id="navbar_oe" class="bgcolor2 fixed-top navbar-expand-md navbar-custom navbar-bright navbar-inner" name="kiosk_hide" data-role="page banner navigation">
-                <!-- Brand and toggle get grouped for better mobile display -->
```

Comment:
Hi @tywrenn  and @bradymiller 
In my opinion, this arrow button is unnecessary, because the navbar already has the toggle to collapse or show.
What do you think?

Reply:
I agree, it seems like an outdated measure for mobile devices that we used to do. You can go ahead and remove it.

Reply:
@tywrenn  update the commit, please take a look.